### ISSUE TITLE
Activity com.google.androidbrowserhelper.launchtwa.LaunchTwaActivity has leaked ServiceConnection

### ISSUE THREAD

**Issue by haitech** [GoogleChrome/android-browser-helper#71]
The library (`com.google.androidbrowserhelper:androidbrowserhelper:1.1.0`) is leaking the `ServiceConnection` when exiting the activity (see below for the stacktrace). I also checked the elder version (1.0.0), this is also leaking.

**How to reproduce:**
- Run e.g. the _TWA-CUSTOM-LAUNCHER_ from the _Demos_
- launch one of the instances, e.g. "_LAUNCH WITH CUSTOM COLORS_"
- exit the customtab, by pressing back or close
- then exit the activity, by pressing back
- In the logcat you will see the activity is leaking the ServiceConnection

**Tested on:**
- Emulator: Pixel 3, API 29, x86
- Samsung Galaxy Tab S3, SM-T820, Android 7.0

**Stacktrace:**
```
E/ActivityThread: Activity com.google.androidbrowserhelper.launchtwa.LaunchTwaActivity has leaked ServiceConnection com.google.androidbrowserhelper.trusted.TwaLauncher$TwaCustomTabsServiceConnection@5605265 that was originally bound here
    android.app.ServiceConnectionLeaked: Activity com.google.androidbrowserhelper.launchtwa.LaunchTwaActivity has leaked ServiceConnection com.google.androidbrowserhelper.trusted.TwaLauncher$TwaCustomTabsServiceConnection@5605265 that was originally bound here
        at android.app.LoadedApk$ServiceDispatcher.<init>(LoadedApk.java:1805)
        at android.app.LoadedApk.getServiceDispatcherCommon(LoadedApk.java:1677)
        at android.app.LoadedApk.getServiceDispatcher(LoadedApk.java:1656)
        at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1721)
        at android.app.ContextImpl.bindService(ContextImpl.java:1650)
        at android.content.ContextWrapper.bindService(ContextWrapper.java:705)
        at androidx.browser.customtabs.CustomTabsClient.bindCustomTabsService(CustomTabsClient.java:76)
        at com.google.androidbrowserhelper.trusted.TwaLauncher.launchTwa(TwaLauncher.java:205)
        at com.google.androidbrowserhelper.trusted.TwaLauncher.launch(TwaLauncher.java:154)
        at com.google.androidbrowserhelper.trusted.TwaLauncher.launch(TwaLauncher.java:173)
        at com.google.androidbrowserhelper.launchtwa.LaunchTwaActivity.launchWithCustomColors(LaunchTwaActivity.java:67)
        at java.lang.reflect.Method.invoke(Native Method)
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:385)
        at android.view.View.performClick(View.java:7125)
        at android.view.View.performClickInternal(View.java:7102)
        at android.view.View.access$3500(View.java:801)
        at android.view.View$PerformClick.run(View.java:27336)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by khatv911** [comment]
you can try pulling the serviceConnection as a global var and unbind it (for instance, in onDestroy)

**Comment by khatv911** [comment]
@haitech @andreban  TwaLaunchers didn't get disposed in onDestroy. I update the fix in #76

**Comment by haitech** [comment]
Good eyes! Thanks @khatv911

**Comment by khatv911** [comment]
glad to help. PR is merged. can close this now

**Comment by andreban** [comment]
Closed via #76
